README.md

```diff
@@ -15,5 +15,9 @@ Start a development server with
 npm start
 ```
 
+For it to read manga, make sure to set up a `config.tsx` file, with 
```

Comment:
why is the config a .tsx, doesnt matter too much tho

Reply:
Trauma from the state thing when trying to implement the state u did in react native - ill move it back to .ts